docs/modules/ROOT/pages/how-tos/update_maintenance.adoc

```diff
@@ -192,6 +194,15 @@ kubectl --as=cluster-admin -n openshift-machine-config-operator \
   logs -c machine-config-daemon -f "${POD}"
 ----
 
+* If the Machine Config operator fails to drain a node, you may have to force-drain the node:
++
+[source,console]
+----
+oc adm drain <node-name> --delete-emptydir-data --ignore-daemonsets --force --grace-period=0
+----
++
+If manually force-draining the node isn't successfull, check which pods are still running on the node with `oc describe node <node-name>` and force delete the pods. The Machine Config operator should then be able to continue with the node upgrades. Depending on what's blocking the drain, these steps may have to be repeated for several nodes.
```

Comment:
One sentence per line.

I'd propose to simply use `oc --as=cluster-admin adm drain --ignore-daemonsets --delete-emptydir-data --force --grace-period=0` to force drain nodes instead of manually finding stuck pods and force deleting them.

Reply:
From experience: If a PDB prevents a pod from being deleted, the drain won't work even with `--force`, only manually deleting the pod allows the operator to continue

Reply:
Ah, good point, I forgot about that.